find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
```

Comment:
I'm not sure, but maybe "return" be more appropriate than "puts"? Otherwise, the code will continue to run after the print statement and then try to return nil

Reply:
I think it's great idea, thank you for pointing it out.